seed/static/seed/js/controllers/data_upload_modal_controller.js

```diff
@@ -244,6 +251,9 @@ angular.module('BE.seed.controller.data_upload_modal', [])
         $scope.uploader.in_progress = false;
         $scope.uploader.progress = 0;
 
+        var modal_element = angular.element(document.getElementsByClassName('modal-dialog'));
```

Comment:
Given this `modal-dialog` element is created by the parent controller, I took this approach to showing a bigger modal only for Meter uploads. Let me know if there was a better way to go about it.

Reply:
@axelstudios - any comments on this?

Reply:
Good enough for now - `$uibModal` has very limited options (and no access to the host element), and because modals are direct children of the `<body>` element it's a pain to have dynamic styles for them.  If we feel like addressing this in the future my suggestion would be to move the `ng-app="BE.seed"` directive to the `<html>` element and create a service to manage classes that apply to `<body>`.  That way you could create a style like `body.modal-lg .modal-dialog` that's set with ngClass on the body from a service.